src/scripts/main.js

```diff
@@ -1 +1,18 @@
 'use strict';
+
+const largeImage = document.getElementById('largeImg');
+const gallery = document.querySelector('.gallery');
+
+gallery.addEventListener('click', () => {
+  event.preventDefault();
+
+  const target = event.target;
+
+  if (target.tagName === 'IMG') {
```

Comment:
Currently, the code only checks if the clicked element is an `IMG`. The task requires handling clicks on both `img` and `a` elements. Consider checking if `target.tagName` is either 'IMG' or 'A'.